affiliates/romania.swagger.json

```diff
@@ -0,0 +1,517 @@
+              },
+              "application/xml": {
+                "schema": {
+                  "$ref": "#/components/schemas/Lead"
+                }
+              }
+            }
+          },
+          "responses": {
+            "200": {
+              "description": "Success",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/LeadResponse"
+                  }
+                },
+                "application/xml": {
+                  "schema": {
+                    "$ref": "#/components/schemas/LeadResponse"
+                  }
+                }
+              }
+            },
+            "401": {
+              "description": "Unauthorized",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/UnauthorizedResponse"
+                  }
+                },
+                "application/xml": {
+                  "schema": {
+                    "$ref": "#/components/schemas/UnauthorizedResponse"
+                  }
+                }
+              }
+            },
+            "422": {
+              "description": "Validation error",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/ValidationErrors"
+                  }
+                },
+                "application/xml": {
+                  "schema": {
+                    "$ref": "#/components/schemas/ValidationErrors"
+                  }
+                }
+              }
+            }
+          },
+          "security": [
+            {
+              "oAuth": [
+                "allow_custom_pingtree"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "components": {
+      "securitySchemes": {
+        "oAuth": {
+          "type": "oauth2",
+          "flows": {
+            "clientCredentials": {
+              "tokenUrl": "/token",
+              "scopes": {
+                "allow_custom_pingtree": "",
+                "get_country_price_list": ""
+              }
+            }
+          }
+        },
+        "bearerAuth": {
+          "type": "http",
+          "scheme": "bearer"
+        }
+      },
+      "schemas": {
+        "Agreements": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/components/schemas/Agreement"
+          },
+          "example": {
+            "terms_of_service": true
+          }
+        },
+        "Agreement": {
+          "type": "boolean"
+        },
+        "Customer": {
+          "required": [
+            "email",
+            "first_name",
+            "last_name",
+            "personal_id",
+            "phone",
+            "agreements"
+          ],
+          "type": "object",
+          "properties": {
+            "first_name": {
```

Comment:
Should add regex format for "first_name" and "last_name"

Reply:
added!